test/unit/spec/nbextensions/appCell2/widgets/appCellWidget-spec.js

```diff
@@ -41,6 +43,8 @@ define([
         execMessage: '[data-element="execMessage"]',
     };
 
+    const { cssCellType, stateMessages } = AppCellWidget;
```

Comment:
import the state messages and use `cssCellType` as the css var name instead of `cellType`